sites/public/src/components/listings/ListingsCombined.module.scss

```diff
@@ -0,0 +1,45 @@
+.listings-combined {
+  // This is a not-ideal way to do "fill window minus header+footer" however I can't find another way to do this.
+  // TODO: update header+footer to a not-magic number
+  --listings-component-height: calc(100vh - 275px);
+  --listings-component-half-height: calc((100vh - 275px) / 2);
```

Comment:
(nit) can this just be `calc(var(--listings-component-height) / 2)`?

Reply:
Yep, done